apps/evm-bridge/src/App.tsx

```diff
@@ -4,10 +4,52 @@
 import { Header, Bridge, Footer } from './components';
 import { DirectionalArrowsSvg } from './components/svgs/DirectionalArrows';
 import { FaucetButton } from './components/FaucetButton';
+import { useEffect, useRef } from 'react';
+import { useCurrentAccount, useCurrentWallet } from '@iota/dapp-kit';
+import { useAccount } from 'wagmi';
+import { ampli, initAmplitude } from './shared/analytics';
 
 const TITLE = 'Seamlessly transfer funds between IOTA & IOTA EVM';
 
 export default function App() {
+    const l1Account = useCurrentAccount();
+    const l1Wallet = useCurrentWallet();
+    const l2Account = useAccount();
+
+    const l1TrackedRef = useRef(false);
+    const l2TrackedRef = useRef(false);
```

Comment:
What do we need these refs for? useEffects only rerun if the deps actually change